styles/AdvancedEditor.css

```diff
@@ -173,11 +173,19 @@
   text-align: left;
 }
 
+.ProseMirror .box-element .block-section_title h1.empty-node[data-placeholder]::before {
+  font-style: italic !important;
+  font-weight: 400 !important;
+  line-height: 24px; 
+  letter-spacing: -0.369px;
+  font-size: 13.5pt ;
```

Comment:
all empty section titles nested or main will have the same font size! that should be for main section

Reply:
As discussed in yesterday's refinement meeting, the default section title, "box title," should be larger than the nested "subsection heading."